GoogleForCreators/web-stories-wp
# Code Quality: Remove storyAnimTool and associated code

## Context

The story animation tool was created for internal use to prototype animations that weren't available to users but necessary for templates. It's been deprecated and is messing with some of the dependency relationships as we pull things out into separate packages.

## Summary

Removes `StoryAnimTool` associated code. 

## Relevant Technical Choices



## To-do



## User-facing changes

No user facing changes 

## Testing Instructions

Now you won't see the story animation tool if you navigate to it: 

`.../wp-admin/edit.php?post_type=web-story&page=stories-dashboard#/story-anim-tool`

Quick pass at regression for the dashboard, I removed a route that's it.


### QA






This PR can be tested by following these steps:

1.

### UAT







This PR can be tested by following these steps:

1.

## Reviews

### Does this PR have a security-related impact?

No

### Does this PR change what data or activity we track or use?

No

### Does this PR have a legal-related impact?

No

## Checklist




---



Fixes #6959
